### DISCUSSION TITLE
How are the diagrams as seen in the README.md created?

### DISCUSSION THREAD

**Discussion by ghost** [giteshnxtlvl/cook#48] [Q&A] [answered]
I've been exploring the repository and noticed the clean and informative diagrams (like the command syntax diagrams). I was wondering what tool or method you used to create these diagrams.

Are they generated with a specific design tool, markdown syntax, or any other approach?

Any pointers or recommendations would be greatly appreciated!

Thanks in advance for your help!

**Comment by glitchedgitz** [comment, marked as answer]
Hey, Those are custom so no specific tool or markdown. I used [figma](https://www.figma.com/) to create those.
